### DISCUSSION TITLE
Is there a way to use user-lever context without auth? Or some other solution for emulate this?

### DISCUSSION THREAD

**Discussion by junioraze** [h2oai/wave#2298] [Q&A]
I hava a iframe app inside my client app, the iframe receive an query arguments wraped in the client app. That queryarg is necessary for show the correct data. All works for one access, but if two or more clients enter the app, the same starts to mix the contexts. So to solve this i need a way to based on my query argument create some functionality like the user-level context in q.user to correct handle my data with multiusers. But i dont know how to do it. Thx for the help..

**Comment by mturoci** [comment]
cc @marek-mihok
